queries.sql

```diff
@@ -7,3 +7,42 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.2;
+
+/* transaction*/
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+/* transaction*/
+BEGIN;
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+COMMIT;
+
```

Comment:
- [ ] Please verify the change was made after the update and rollback as requested in the project requirements.
  - Hint: `SELECT species from animals;`

Reply:
Hello,
Kindly view the screenshot attached to the PR. It shows the full table including the species column. This clearly exhibits the changes before and after they were made.

Reply:
@thecodechaser It is all there. kindly view the documents thoroughly before make your reviews.

Reply:
@Ceemos96 No it's not there, Please review your file.
After the update, add commands to verify that the change was made. I already provided you with a hint

Reply:
Here is a copy of the attached screenshot in the PR description. 
It clearly shows the animals table including the species column,  it also shows that all rows have the 'unspecified' value. Futhermore, it shows that i have rolled back the changes as required by the project.

![Screenshot (126)](https://user-images.githubusercontent.com/89014423/207620722-e3020fc6-c662-4511-9582-7e595c9e78ef.png)